main.js

```diff
@@ -0,0 +1,85 @@
+var newPostTemplate = function(postText,nameText) {
+  var postRemoveButton = '<button type="button" class="btn btn-link rem-post-btn">[X]</button>';
+  var postCommentButton = '<button type="button" class="btn btn-link view-comms-btn">[comments]</button>';
+  var commentFormDisplay = '<div class="toggler hide-element"><div class="comments"></div><form><div class="form-group"><input type="text" class="form-control comment-post comment-width-adj" id="comment-post" placeholder="Comment Text" /></div><div class="form-group"><input type="text" class="form-control comment-name comment-width-adj" id="comment-name" placeholder="Your Name" /></div></form>';
+  var commentSubmitButton = '<button type="button" class="btn btn-info submit-comm-btn">Submit Comment</button></div>';
+  var combinedCommentDisplay = commentFormDisplay + commentSubmitButton;
+  var insertPost = '<div class="new-pst"><h4><p>'+ postRemoveButton + postCommentButton + '<strong>'+ postText + '</strong><small> - posted by: ' + nameText + '</small></p></h4>' + combinedCommentDisplay + '<hr></div>';
+
+  $('.posts').append(insertPost);
+
+  initPostRemoveListener();
+  initCommentToggleListener();
+  initCommentSubmitListener();
+};
+
+var newCommentTemplate = function(commentText,commentName,matchCommentToPost) {
+  var commentRemoveButton = '<button type="button" class="btn btn-link rem-comm-btn">[x]</button>';
+  var insertComment = '<div class="new-com"><h5><p>' + commentRemoveButton + commentText +'<small> - posted by: ' + commentName + '</small></p></h5></div>';
+
+  matchCommentToPost.append(insertComment);
+
+  initCommentRemoveListener();
+};
+
+var initPostSubmitListener = function () {
+  $('.submit-post-btn').click(function() {
+    var postTextInput = $('#input-post').val();
+    var postNameInput = $('#input-name').val();
+
+    if (postTextInput == '' || postNameInput == '') {
+      return alert('Please submit both your post and name.');
+    };
+
+    for(i=0; i < $('form').length; i++) {
+      $('form')[i].reset();
+    };
+
+    newPostTemplate(postTextInput,postNameInput);
+  });
+};
+
+var initCommentSubmitListener = function() {
+  $('.submit-comm-btn').off('click');
+  $('.submit-comm-btn').on('click', function() {
+    var commentTextInput = $(this).closest('.toggler').find('#comment-post').first().val();
+    var commentNameInput = $(this).closest('.toggler').find('#comment-name').first().val();
```

Comment:
Chaining these jQuery methods works perfectly well, but it makes your code less resilient (what happens if the DOM changes?). I'd recommend considering other approaches.